bmm/src/main/java/org/openehr/bmm/core/BmmClass.java

```diff
@@ -425,7 +446,7 @@ protected void populateTarget(BmmClass source, BmmClass target) {
 
     protected void handleFlattenedProperty(BmmProperty property, BmmClass target) {
         if (target.hasPropertyWithName(property.getName())) {
-            throw new RuntimeException("Property with name " + property.getName() + " already defined in type " + target.getName() + " or one of its ancestors");
+            //this is fine, it has been validated to be conformant and just overrides the old property
```

Comment:
Since this case doesn't do anything anymore, it would be a little bit cleaner to reverse this if statement